src/Kernels/KernelTensors.jl

```diff
@@ -42,16 +43,16 @@ struct CoalescenceTensor{FT} <: KernelTensor{FT}
     "polynomial order of the tensor"
     r::Int
     "collision-coalesence rate matrix"
-    c::Array{FT}
+    c::AbstractArray{FT}
```

Comment:
```suggestion
    c::StaticArrays.SVector{N,FT}
```
and make `N` a type parameter above (if this is an SVector)

Using abstract container types is generally a performance pitfall (https://docs.julialang.org/en/v1/manual/performance-tips/#Avoid-fields-with-abstract-type)